VAMobile/src/components/VAIcon/VAIcon.tsx

```diff
@@ -223,12 +225,32 @@ const VAIcon: FC<VAIconProps> = ({ name, width, height, fill, fill2, stroke, pre
     return <></>
   }
 
-  if (width && isFinite(width)) {
-    iconProps = Object.assign({}, iconProps, { width: preventScaling ? width : fs(width) })
+  // Default to scaled dimensions
+  const scaledWidth = width && fs(width)
+  const scaledHeight = height && fs(height)
```

Comment:
Doesn't need to exist as a const, only used once.

Reply:
Removed